samples/snippets/firewall/create.py

```diff
@@ -0,0 +1,74 @@
+#  Copyright 2022 Google LLC
+#
+#  Licensed under the Apache License, Version 2.0 (the "License");
+#  you may not use this file except in compliance with the License.
+#  You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+#  Unless required by applicable law or agreed to in writing, software
+#  distributed under the License is distributed on an "AS IS" BASIS,
+#  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+#  See the License for the specific language governing permissions and
+#  limitations under the License.
+# flake8: noqa
+
+
+# This file is automatically generated. Please do not modify it directly.
+# Find the relevant recipe file in the samples/recipes or samples/ingredients
+# directory and apply your changes there.
+
+
+# [START compute_firewall_create]
+from google.cloud import compute_v1
+
+
+def create_firewall_rule(
```

Comment:
The tag `compute_firewall_create` doesn't match `create_firewall_rule` . Is that expected?

```suggestion
def compute_firewall_create(
```

Reply:
Yes, that is intentional. Tag needs to start with a product, in this case `compute`, while the method names usually start with a verb. The region tags don't have to match the function names.